tests/test_searcher.py

```diff
@@ -67,11 +69,17 @@ class TestPerformance:
     python3 -m pytest tests/test_searcher.py::TestPerformance -sv --runslow
     """
 
+    @pytest.fixture
+    def request_fixture(self, request):
```

Comment:
sorry being naive here - where does the input arg `request` come from?

Am looking at the tests below, you can call the arg by `request_fixture.node.name`, not quite clear to here where does the node.name come from

Reply:
I have no idea how this works, just that it works